src/pages/productListing/ProductListing.jsx

```diff
@@ -0,0 +1,166 @@
+import Card from "../components/card/card";
+import React from 'react';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+export default function ProductListing({ products }) {
+
+
+  const [item,setItem] = useState([]);
+
+  useEffect(()=>{
+    
+
+    axios.get('/api/products')
+    .then((response)=>{
+      setItem(response.data.products);
+
+    },
+    (error)=>{
+      console.log(error);
+
```

Comment:
try to use try..catch format for good practise - (Personal Suggestion you can use this later after finishing e-comm)